### ISSUE TITLE
Metadata timestamp not being tested

### ISSUE THREAD

**Issue by shaeussler** [CycloneDX/cyclonedx-dotnet#815]
I miss this code snippet in the Runner.cs:
```
            // Automatically generate a timestamp if none is provided with the metadata
            if (bom.Metadata.Timestamp == null)
            {
                bom.Metadata.Timestamp = DateTime.UtcNow;
            }
```

**Comment by mtsfoni** [comment]
Thanks

It will be back soon

**Comment by mtsfoni** [comment]
Fixed in 3.0.3

Further to do: Add Test-cases to check behaviour when metadata is or isn't imported.

**Comment by shaeussler** [comment]
Thanks for the fast fix.
